scripts/in_container/configure_environment.sh

```diff
@@ -23,6 +23,7 @@ readonly TMUX_CONF_FILE=".tmux.conf"
 
 if [[ -d "${FILES_DIR}" ]]; then
     export AIRFLOW__CORE__DAGS_FOLDER="/files/dags"
+    export AIRFLOW__CORE__PLUGINS_FOLDER='/files/plugins'
```

Comment:
This makes some tests to fail because they rely on the folder to point to `tests/plugins`, but I can’t find where this is set 😞

Reply:
I think it's set from - `airflow/airflow/configuration.py` and `airflow/airflow/config_templates/default_test.cfg`

Reply:
Ah there’s a `load_test_config`, makes sense, good find.

> should be one level up

Or they should be in the `tests` directory. But code around this was designed _7 years ago_ so I’m not going to waste time complaining 😉  We can probably improve it if we want to…

Reply:
> Or they should be in the tests directory. But code around this was designed 7 years ago so I’m not going to waste time complaining 😉 We can probably improve it if we want to…

Yeah. It was LOOONG before my time and I always found it cumbersome, but never gotten to fix it. 

But we've moved to pytest and actually, it could be something that we can turn into auto-used pytest_fixtures to set the right environment for alll tests, I believe. Before moving to pytest, this "load_test_config" thingie was not really easy to do, but with conftest.py and fixtures, all that it does, can I think be done there.

I think this would inflate the scope of this change quite a lot, but might be worth it. 

@utkarsharma2 - WDYT?

Reply:
@uranusjr @potiuk Ya, I too think we should improve it. Will extend this PR to cover this requirement.

Reply:
This has been somewhat addressed in #32525.